pkg/artifactory/util.go

```diff
@@ -116,6 +116,17 @@ func getMD5Hash(o interface{}) string {
 	return hex.EncodeToString(hasher.Sum(nil))
 }
 
+var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
```

Comment:
So, this function should really be called `randomString` and please move the selectable characters into function itself. Also note: you can actually do this numerically the letter. Capital `A` is hex `0x40` - to get `B` you simply add 1

These sorts of library functions have to be out there in the wild

Reply:
removed the method `randomSeq`. Instead use `randomInt` as per @alexhung suggestion.